src/foam/lib/json/Outputter.java

```diff
@@ -32,33 +34,43 @@ protected SimpleDateFormat initialValue() {
     }
   };
 
-  protected PrintWriter   writer_;
-  protected OutputterMode mode_;
```

Comment:
Seems like we're ditching the idea of having predefined outputter configurations. Why?